homeassistant/components/light/lcn.py

```diff
@@ -29,7 +29,13 @@
         connection = get_connection(connections, connection_id)
         address_connection = connection.get_address_conn(addr)
 
-        devices.append(LcnOutputLight(config, address_connection))
+        if config[CONF_OUTPUT] in OUTPUT_PORTS:
```

Comment:
Marker: This is the conditional if clause I mentioned above.
Without changing the constant lists to uppercase and exchange the position of the validators, I would have to use config[CONF_OUTPUT].lower() which is inconsistent with fact, that all formatting should be done in the validators.